Std/Data/Rat/Basic.lean

```diff
@@ -34,25 +34,28 @@ instance : Repr Rat where
 
 theorem Rat.den_pos (self : Rat) : 0 < self.den := Nat.pos_of_ne_zero self.den_nz
 
+-- Note: `Rat.normalize` uses `Int.div` internally,
+-- but we may want to refactor to use `/` (`Int.ediv`)
```

Comment:
As you have presumably already discovered, the difference does not matter in this case because it is only used for exact division, where all conventions agree. I am inclined to use `.div` in the implementation and `/` in any definition unfolding simp lemmas and side conditions where possible.